content/xp-arch-framework/interface-integrations/platform-continuity.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Platform Continuity"
+weight: 9
+description: "A guide for how to integrate control planes with DR tools"
+---
+
+Platform continuity is a topic that encompasses everything you need to do to keep your platform up and running. This content is currently focused on providing guidance for how to perform disaster recovery with Crossplane.
+
+## Disaster Recovery
+
+[Velero](https://velero.io/) is a popular open source solution in the Kubernetes ecosystem to safel perform backup and restore operations against Kubernetes cluster resources. We recommend using Velero as the key ingredient for how to do disaster recovery with your Crossplane control planes. We will cover best practices for using Velero in the context of Crossplane.
+
+### In the context of Crossplane
+
+It's important to remember:
+
+1. Crossplane is concerned with ensuring the configuration of your resource. 
+2. Crossplane **isn't concerned** with the _internal_ state of your resources. 
+
+Crossplane is concerned with enforcing two things:
+
+1. the resources you declared exist.
+2. the configuration of those resources matches what you declared. If it doesn't, it will attempt to reconcile to the desired state.
+
+Crossplane **isn't** concerned with the state of what's happening _within_ the resource. For example, consider a CloudSQL instance created with Crossplane using the following manifest:
+
+```yaml
+apiVersion: sql.gcp.upbound.io/v1beta1
+kind: DatabaseInstance
+spec:
+  forProvider:
+    databaseVersion: MYSQL_5_7
+    region: us-central1
+    settings:
+      - diskSize: 20
+        tier: db-f1-micro
+```
+
+Crossplane will ensure whatever instance gets created in GCP has a disk size of 20 GB and that it uses MySQL version 5.7; it will continuously ensure the configuration of the instance matches these parameters. However, Crossplane has no involvement with what goes on _inside_ the CloudSQL instance. If someone was using this database to store records of users for a line-of-business app, Crossplane is oblivious.
```

Comment:
"Disaster Recovery for Crossplane != disaster recovery for your business critical state, like database or cloud storage data."